governator-core/src/main/java/com/netflix/governator/Governator.java

```diff
@@ -73,7 +73,9 @@ protected void configure() {
                 manager.notifyShutdown();
             }
             catch (Exception e2) {
-                
+            }
+            finally {
+                System.exit(-1);
```

Comment:
Just curious, what is the rationale for exiting the jvm? If used as to create an independent LifecycleInjector in a test for example that seems a bit extreme.